src/xunit/xunit_lib_tara.cpp

```diff
@@ -434,15 +434,15 @@ BOOL GetManualExposureValue_Stereo(INT32 *ExposureValue)
 
 
 /*
-  **********************************************************************************************************
+ **********************************************************************************************************
  *  MODULE TYPE	:	LIBRAY API 										*
  *  Name		:	SetManualExposureValue_Stereo					*
  *  Parameter1	:	INT32	(ExposureValue)					    	*
  *  Returns		:	BOOL (TRUE or FALSE)							*
  *  Description	:   Sends the extension unit command to set the manual exposure value to the camera   *
  *					The exposure value ranges from 1 to 1000,000					  			  	  *
-  **********************************************************************************************************
-*/
+ **********************************************************************************************************
+ */
```

Comment:
Add a comment here mentioning what are the supported values and what value should be used for setting auto exposure. @akhileshmalviya